frontend/public/components/namespace.jsx

```diff
@@ -700,7 +700,7 @@ const ProjectLink = ({ project }) => {
 
 export const ProjectsTable = (props) => {
   const { t } = useTranslation();
-  const columns = useProjectsColumns({ showMetrics: false, showActions: false });
+  const columns = useProjectsColumns({ showMetrics: false, showActions: true });
```

Comment:
It should be false.  `ProjectsTable` is used in the Developer perspective, where actions are not needed. 

```suggestion
  const columns = useProjectsColumns({ showMetrics: false, showActions: false });
```